test/gas/custom_gas_price.js

```diff
@@ -0,0 +1,22 @@
+const assert = require("assert");
+const getWeb3 = require("../helpers/web3/getWeb3");
+
+describe("Custom Gas Price", () => {
+  it("should return gas price of 15 when specified as a decimal", async() => {
+    const options = {
+      gasPrice: 15
+    };
+    const { web3 } = await getWeb3(options);
+    const result = await web3.eth.getGasPrice();
+    assert.strictEqual(parseInt(result), 15);
+  });
+
+  it("should return gas price of 15 when specified as hex", async() => {
+    const options = {
+      gasPrice: 0xf
```

Comment:
I would have expected this to be a hex string rather than a hex literal. If you're specifying a hex literal this test is logically equivalent to the previous one

Reply:
`gasPrice` is specified as a hex on line 16. I can update the description to be clearer.

Reply:
@ccowell @ben is just saying that `0xf === 16` whereas `"0xf" !== 16` (strictly). One version tests if ganache can understand a _`Number`_ and the other a hex _`string`_

Reply:
What @davidmurdoch said, but my original comment was that today both tests appear to check if ganache can understand a `Number` and no tests check if ganache can understand a hex-encoded `string`

Reply:
The initial round of refactoring effort focused on cleaning up code and not creating new tests. That test can be added for completeness.

Reply:
Updated.